deploy-manage/security/ip-filtering-ece.md

```diff
@@ -0,0 +1,99 @@
+---
+navigation_title: In ECE
+mapped_pages:
+  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-traffic-filtering-ip.html
+applies_to:
+  deployment:
+    ece: ga
+products:
+  - id: cloud-enterprise
+---
+
+# Manage IP filters in ECE
+
+Filtering by IP address or CIDR block is one of the security layers available in {{ece}}. It allows you to limit how your deployments can be accessed.
+
+You can only configure ingress or inbound IP filters. These restrict access to your deployments from a set of IP addresses or CIDR blocks.
+
+Follow the step described here to set up ingress or inbound IP filters through the Cloud UI.
+
+To learn how traffic filter rules work together, refer to [](ece-filter-rules.md).
+
+To learn how to manage IP filters using the Traffic Filtering API, refer to [](/deploy-manage/security/ec-traffic-filtering-through-the-api.md).
+
+:::{note}
+To learn how to create IP filters for {{ech}} deployments or {{serverless-full}} projects, refer to [](ip-filtering-cloud.md).
+
+To learn how to create IP filters for self-managed clusters or {{eck}} deployments, refer to [](ip-filtering-basic.md).
+:::
+
+## Prerequisites
+
+Make sure your [load balancer](/deploy-manage/deploy/cloud-enterprise/ece-load-balancers.md) handles the `X-Forwarded-For` header appropriately for HTTP requests to prevent IP address spoofing. Make sure the proxy protocol v2 is enabled for HTTP and transport protocols (9243 and 9343).
+
+## Apply an IP filter to a deployment
+
+To apply an IP filter to a deployment, you must first create a rule set at the organization or platform level, and then apply the rule set to your deployment.
+
+### Step 1: Create an IP filter rule set
+
+You can combine any rules into a set, so we recommend that you group rules according to what they allow, and make sure to label them accordingly. Since multiple sets can be applied to a deployment, you can be as granular in your sets as you feel is necessary.
+
+To create a rule set:
+
+1. [Log into the Cloud UI](/deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md).
+2. From the **Platform** menu, select **Security**.
+3. Select **Create filter**.
+4. Select **IP filtering rule set**.
+5. Create your rule set, providing a meaningful name and description.
+6. Select the region for the rule set.
+7. Select if this rule set should be automatically attached to new deployments.
+
+    ::::{note}
+    Each rule set is bound to a particular region and can be only assigned to deployments in the same region.
+    ::::
```

Comment:
Same as before with ECE regions, not sure if we provide that concept across ECE docs.